data-lineage/src/main/java/com/wipro/ats/bdre/lineage/LineageProcessor.java

```diff
@@ -44,25 +66,40 @@ public void execute(String wholeQuery, String processId, String instanceExecId)
 					int commentStart = query.indexOf("--");
 					int commentEnd = query.indexOf("\n", commentStart+2);
 					String comment = query.substring(commentStart, commentEnd);
-					System.out.println("comment found = " + comment);
+					LOGGER.debug("comment found = " + comment);
 					query = query.replaceAll(comment, " ");
 				}
 				// remove line breaks
 				query = query.replaceAll("[\\r\\n]", " ").trim();
-				System.out.println("\nLineageProcessor: Processing query # " + (i + 1) + ": \"" + query + "\"");
+				LOGGER.debug("\nLineageProcessor: Processing query # " + (i + 1) + ": \"" + query + "\"");
 
 				if (query.toLowerCase().startsWith("use")) {
 					// ignore 'use db' line for lineage
 					// use last split to determine the db
 					String[] splits = query.split(" ");
 					defaultHiveDbName = splits[splits.length-1].toUpperCase();
-					System.out.println("DefaulHiveDbName is set to " + defaultHiveDbName);
+					LOGGER.debug("DefaulHiveDbName is set to " + defaultHiveDbName);
 
-				} else if (query.toLowerCase().startsWith("insert")
+				}
+				//Wrting queries having "use" "insert" & "update" clauses to the DB to be later used by DotGen class
+				if (query.toLowerCase().startsWith("use")
+						|| query.toLowerCase().startsWith("insert")
 						|| query.toLowerCase().startsWith("select")) {              // process only insert and select statements
+					//insert the query into DB using its DAO
+					LineageQuery lineageQuery = new LineageQuery();
+					lineageQuery.setQueryString(query);
+					lineageQuery.setInstanceExecId(Long.parseLong(instanceExecId));
+					lineageQuery.setProcessId(Integer.parseInt(processId));
+					lineageQuery.setCreateTs(new Date());
+					id = UUID.randomUUID().toString();
+					lineageQuery.setQueryId(id);
+					LineageQueryType lineageQueryType = new LineageQueryType(1, "HIVE");
+					lineageQuery.setLineageQueryType(lineageQueryType);
+					lineageQueryDAO.insert(lineageQuery);
 
 					// LineageMain.main(new String[]{query, processId, instanceExecId});
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)